packages/shared/src/components/modals/common.tsx

```diff
@@ -159,6 +159,13 @@ const SlackIntegrationModal = dynamic(
     ),
 );
 
+const ReportSourceModal = dynamic(
+  () =>
+    import(
+      /* webpackChunkName: "reportSquadModal" */ './report/ReportSourceModal'
```

Comment:
Probably meant this?
```suggestion
      /* webpackChunkName: "reportSourceModal" */ './report/ReportSourceModal'
```